nbbuild/build.xml

```diff
@@ -2044,6 +2044,19 @@ It is possible to use -Ddebug.port=3234 -Ddebug.pause=y to start the system in d
                 <exclude name="projectuiapi/test/unit/src/org/netbeans/modules/project/uiapi/data/*.txt" /> <!--test data-->
                 <exclude name="projectui/src/org/netbeans/modules/project/ui/resources/*-license.txt" /> <!--licenses-->
                 <exclude name="projectui/src/org/netbeans/modules/project/ui/resources/license-default.txt" /> <!--licenses-->
+                <exclude name="css.prep/test/qa-functional/data/css_prep/public_html/cc.less" /> <!--simple file for testing-->
+                <exclude name="css.prep/test/qa-functional/data/css_prep/public_html/cc.scss" /> <!--simple file for testing-->
```

Comment:
Maybe better to exclude `css.prep/test/qa-functional/data/css_prep/public_html/**`
and `css.prep/test/unit/data/testProject/public_html/*` ?